Stop cargo from attempting to update it, we're already unpacking same revision.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -72,4 +72,2 @@
 inherits = "release"
 
-[patch.crates-io]
-naga = { git = "https://github.com/gfx-rs/naga", rev = "1be8024" }
\ No newline at end of file
